src/validation/loginValidation.js

```diff
@@ -0,0 +1,8 @@
+import Joi from '@hapi/joi';
+
+const loginSchema = Joi.object({
+  email: Joi.string().email({ minDomainSegments: 2, tlds: { allow: ['com', 'net'] } }),
+  password: Joi.string().pattern(new RegExp('^[a-zA-Z0-9@]{4,16}$')),
```

Comment:
@EugeneMunya as @Niyonsengaeric , the password is validated while signing up no need to validate it again while logging in.

Reply:
Its true but I wanted not everyone entered  any random password I have to hit  the database for looking whether is match or is not. but I going to change it, its not necessary.